Revit_Adapter/AdapterActions/Pull.cs

```diff
@@ -61,11 +61,11 @@ public override IEnumerable<object> Pull(IRequest request, PullType pullType = P
             if (!CheckConnection())
                 return new List<object>();
             
-            if (!(request is FilterRequest))
+            if (!(request is IRequest))
                 return new List<object>();
```

Comment:
This will never be true. You can remove the check.

Reply:
Fixed